micro_sam/sam_annotator/annotator_3d.py

```diff
@@ -104,7 +104,7 @@ def _segment_volume_for_auto_segmentation(
 
 
 @magicgui(
-    call_button="Segment Volume [V]",
+    call_button="Segment All Slices [Shift-S]",
```

Comment:
I've changed the language here from "Segment Volume" to "Segment All Slices" because I felt that might make it easier to remember the keybindings. I hope it is unlikely to cause confusion for users, what do you think?

Reply:
Yes, I think that's fine.